axelrod/tournament.py

```diff
@@ -101,7 +101,7 @@ def play(self, build_results=True, filename=None,
             self.progress_bar = tqdm.tqdm(total=len(self.match_generator),
                                           desc="Playing matches")
 
-        if on_windows and (filename is None):
+        if on_windows and (filename is None):  # pragma: no cover
```

Comment:
Some of these, we can test for (on appveyor for example using skips based on the OS) but we're not running coveralls through them (we get our coverage report sent to coveralls from travis). Is this the best way to deal with it? (I think it might be but worth making sure).